keras_hub/src/models/gemma3/gemma3_backbone.py

```diff
@@ -339,6 +334,14 @@ def get_config(self):
         )
         return config
 
+    def get_lora_target_names(self):
+        target_names = super().get_lora_target_names()
+
+        # Add these for `Gemma3VITAttention`.
+        if not self.text_only_model:
+            target_names += ["query_proj", "value_proj"]
```

Comment:
Does it hurt to just always leave these as part of the targets? They just won't match right?

Reply:
Hmmm, yeah. But let's keep the if condition for clarity